windows-apps-src/design/basics/navigation-basics.md

```diff
@@ -172,20 +172,31 @@ With few exceptions, any app that has multiple pages uses a frame. Typically, an
         ![tabs and pivot image](images/nav/thumbnail-tabs-pivot.svg)
     :::column-end:::
 	:::column span="2":::
-        [**Top navigation and tabs**](../controls-and-patterns/navigationview.md)
+        [**Top navigation**](../controls-and-patterns/navigationview.md)
 
-Displays a horizontal list of links to pages at the same level. The [NavigationView](../controls-and-patterns/navigationview.md) control implements the top navigation and tabs patterns.
+Displays a horizontal list of links to pages at the same level. The [NavigationView](../controls-and-patterns/navigationview.md) control implements the top navigation pattern.
         
 Use top navigation when:
 
 - You want to show all navigation options on the screen.
 - You desire more space for your app's content.
 - Icons cannot clearly describe your navigation categories.
-		
+
+:::row-end:::
+
+:::row:::
+    :::column:::
+        ![tabs and pivot image](images/nav/thumbnail-tabs-pivot.svg)
+    :::column-end:::
+	:::column span="2":::
+        [**Tabs**](../controls-and-patterns/tab-view.md)
+
+Displays a horizontal set of tabs and their respective content. The [TabView](../controls-and-patterns/tab-view.md) control is useful for displaying several pages (or documents) while giving the user the capability to rearrange, open, or close tabs.
+	
 Use tabs when:
 
-- You want to preserve navigation history and page state.
-- You expect users to switch between tabs frequently.
```

Comment:
Isn't this still valid point?

Reply:
These are still true and valid, but I don't think they are major differentiators between TabView and NavigationView. The heart of TabView is about adding/moving/removing tabs, not so much about the user clicking often on different tabs. I can put them back in if you feel strongly.